### PR TITLE
Testing policies. Review and refresh.

### PR DESCRIPTION
**Pull request by jhkrug**
Part one, restructure and move to be immediately after 'Writing policies'. More logical placing than under Explanations.

### CODE REVIEW

**Review by flavio — APPROVED** [review]
I like the change. One minor note, I think it would be nice to have the `Policy testing personas` be the page shown when the user clicks on the `Testing policies` entry on the sidebar (same effect as when the user clicks on the `how-tos` -> `Operator Manual` - see [here](https://docs.kubewarden.io/operator-manual/intro)).

**Comment by jhkrug** [comment]
> I like the change. One minor note, I think it would be nice to have the `Policy testing personas` be the page shown when the user clicks on the `Testing policies` entry on the sidebar (same effect as when the user clicks on the `how-tos` -> `Operator Manual` - see [here](https://docs.kubewarden.io/operator-manual/intro)).

Hi @flavio . Yes, I'll do that tomorrow.

File: docs/testing-policies/02-policy-authors.md [hunk 1]
```diff
@@ -1,60 +1,46 @@
...
+Policy authors can use their favorite development environments, tools and, testing frameworks to verify their development.
 
-Policy authors can leverage the testing frameworks and tools of their language
-of choice to verify the behaviour of their policies.
+These three Kubewarden policies provide an example written in each of [Rust](/writing-policies/rust/01-intro-rust.md), [Go](/writing-policies/go/01-intro-go.md) and [AssemblyScript](https://www.assemblyscript.org/) respectively:
 
-As an example, you can take a look at these Kubewarden policies:
+- [psp-apparmor](https://github.com/kubewarden/psp-apparmor)
+- [ingress-policy](https://github.com/kubewarden/ingress-policy)
+- [pod-privileged-policy](https://github.com/kubewarden/pod-privileged-policy)
```
Comment (jvanz) [inline docs/testing-policies/02-policy-authors.md:17]: This policy has been rewritten with rust in a recent past. As far as I can remember, we do not have assembly script policies anymore.

**Review by jvanz — CHANGES_REQUESTED** [review]
LGTM. I just have a fix considering recent change in a policy mentioned in the documentation.

Furthermore, I really like the structure to hide the content of file in the docs.

File: docs/testing-policies/02-policy-authors.md (hunk 1 above)
Comment (jhkrug) [reply docs/testing-policies/02-policy-authors.md:17]: Thanks @jvanz . Ok, so to be clear. KW supports AssembyScript as it produces Wasm binaries. But, we don't have any AS language policies? Are you aware of any at all out there? 

Moving forward, just reduce to talking about two, one in Rust and one in Go?

Comment (jhkrug) [reply docs/testing-policies/02-policy-authors.md]: Removed reference to AS for now.

**Review by flavio — APPROVED** [review]
I love it, ship it!

**Comment by jhkrug** [comment]
Thanks @flavio. It's ready to ship, Nothing really changed with the last commit. I just have a new tool which helped with some further language simplification. I'll press the green button shortly.

**Review by viccuad — APPROVED** [review]
LGTM! Nice!

**Pull request merged**